m4opt/missions/_ultrasat.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from regions import RectangleSkyRegion
+from synphot import Gaussian1D, SpectralElement
+
+from .. import skygrid
+from ..constraints import (
+    EarthLimbConstraint,
+    MoonSeparationConstraint,
+    SunSeparationConstraint,
+)
+from ..models import Detector
+from ..models.background import GalacticBackground, ZodiacalBackground
+from ..orbit import TLE
+from ..utils.dynamics import Slew
+from ._core import Mission
+
+ultrasat = Mission(
+    name="ultrasat",
+    fov=RectangleSkyRegion(
+        center=SkyCoord(0 * u.deg, 0 * u.deg), width=14.28 * u.deg, height=14.28 * u.deg
+    ),
+    constraints=[
+        EarthLimbConstraint(28 * u.deg),
+        SunSeparationConstraint(46 * u.deg),
+        MoonSeparationConstraint(23 * u.deg),
+        # GalacticLatitudeConstraint(10 * u.deg),
+    ],
+    detector=Detector(
+        # Total number of pixels
+        npix=89.9e6,
```

Comment:
The `npix` property is supposed to be the effective number of pixels in the photometric source extraction aperture. This number can't be correct: the light from a point source isn't spread over nearly 100 million pixels, is it?

Reply:
Here, I used the total number from the Table 1 of  [arXiv:2304.14482](https://arxiv.org/pdf/2304.14482). 
I will double-check other references .

Reply:
Is the Ultrasat PSF Nyquist-sampled?

Reply:
No. What I found in the paper is: the Full Width at Half Maximum (FWHM) of the PSF for a black-body source in the central 170 deg^2 is 8.3 arcseconds. The pixel scale is 5.4 arcseconds / pixel.
But I don't know if the npix should be ,   FWHM / pixel scale ~ 1.54 pixels .

Reply:
That's slightly undersampled. 4 pi is appropriate for critical sampling. I think 4 pi is a fine number to start with for now.